tedana/tests/test_model_fit_dependence_metrics.py

```diff
@@ -0,0 +1,92 @@
+"""
+Tests for tedana.model.fit
+"""
+
+import numpy as np
+import pytest
+
+from tedana.model import fit
+
+
+def test_break_dependence_metrics():
+    """
+    Ensure that dependence_metrics fails when input data do not have the right
+    shapes.
```

Comment:
I'm not sure if we have formatting guidelines for tests; I don't recall seeing any in the linked NumPy style guide either, so I could be off base. 
Could we put brief comments indicating what these tests are checking for? The formatting makes it somewhat difficult to me (and it might just be me) to tell what is actually being tested at each assert.